acquia/acquia_cms
# ACMS-1389: Content type templates for Article, Person, Place, Event, Page

**Motivation**
Fixes #ACMS-1389

**Proposed changes**
- Rewrite Image field to include DAM media.
- Updated content type templates for Article, Person, Place, Event, Page for Media(CORE, DAM).
- Added new view mode 'Referenced Image' for Place and Person content type.
- Updated Article, Event Site studio templates, default view mode.
- Added update hook to update view mode for Article, Event, Person, Place.

**Alternatives considered**
NA

**Testing steps**
- Use this branch
- Install site using acms site:install
- Install ACMS DAM module drush in acquia_cms_dam
- Login into application
- Create node using any content type
- Select DAM media for image in first node
- Select CORE media for image in second node
- Verify images rendering fine on both nodes

**Merge requirements**
